README.md

```diff
@@ -48,14 +48,23 @@ higher-level APIs:
   workloads should be spread across different clusters similar to how a human
   operator would
 
-# Guides
+## Features
 
-## User Guide
+| Feature | Maturity |
+|---------|----------|
+| Generate Federation APIs without writing code | Alpha |
```

Comment:
@pmorie would you please link each feature to its related document?

Reply:
@gyliu513 what type of document are you thinking?

Reply:
Like:

RSP: https://github.com/kubernetes-sigs/federation-v2/blob/master/docs/userguide.md#replicaschedulingpreference
Multicluster DNS: https://github.com/kubernetes-sigs/federation-v2/blob/master/docs/servicedns-with-externaldns.md
Multicluster Ingress: https://github.com/kubernetes-sigs/federation-v2/blob/master/docs/ingressdns-with-externaldns.md
Generate Federation APIs without writing code: https://github.com/kubernetes-sigs/federation-v2/blob/master/docs/userguide.md#enabling-federation-of-an-api-type

Reply:
ah ok